Even though libcurl.so explicitly links to its dependencies, libcurl.pc keeps them under Libs.private. This patch takes that variable into account, adding --static-libs to the curl-config command.

Signed-off-by: Lucas C. Villa Real <lucasvr@gobolinux.org>
--- pycurl-7.16.4/setup.py.orig	2007-12-09 21:28:01.000000000 -0800
+++ pycurl-7.16.4/setup.py	2007-12-09 21:31:22.000000000 -0800
@@ -96,7 +96,7 @@ else:
                 include_dirs.append(e[2:])
         else:
             extra_compile_args.append(e)
-    for e in split_quoted(os.popen("'%s' --libs" % CURL_CONFIG).read()):
+    for e in split_quoted(os.popen("'%s' --libs --static-libs" % CURL_CONFIG).read()):
         if e[:2] == "-l":
             libraries.append(e[2:])
             if e[2:] == 'ssl':
